yarn.lock

```diff
@@ -8034,13 +8034,6 @@ detect-conflict@^1.0.0:
   resolved "https://registry.yarnpkg.com/detect-conflict/-/detect-conflict-1.0.1.tgz#088657a66a961c05019db7c4230883b1c6b4176e"
```

Comment:
how come this file is updated? We didnt update any package.json dependencies did we?

Reply:
Re-running `yarn` again

Reply:
You're correct. I have no idea why this lockfile was updated.

Reply:
Thanks for the tip!